src/main/java/bot/newsaggregator/service/TelegramBot.java

```diff
@@ -0,0 +1,113 @@
+package bot.newsaggregator.service;
+
+import bot.newsaggregator.config.BotConfig;
+import bot.newsaggregator.entity.User;
+import bot.newsaggregator.model.UserRepository;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import org.telegram.telegrambots.bots.TelegramLongPollingBot;
+import org.telegram.telegrambots.meta.api.methods.commands.SetMyCommands;
+import org.telegram.telegrambots.meta.api.methods.send.SendMessage;
+import org.telegram.telegrambots.meta.api.objects.Chat;
+import org.telegram.telegrambots.meta.api.objects.Message;
+import org.telegram.telegrambots.meta.api.objects.Update;
+import org.telegram.telegrambots.meta.api.objects.commands.BotCommand;
+import org.telegram.telegrambots.meta.api.objects.commands.scope.BotCommandScopeDefault;
+import org.telegram.telegrambots.meta.exceptions.TelegramApiException;
+
+import java.sql.Timestamp;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+@Slf4j
+@Component
+public class TelegramBot extends TelegramLongPollingBot {
+    @Autowired
+    private UserRepository userRepository;
+    private final BotConfig config;
+    static final String HELP_TEXT = "This bot is created for news aggregation.\n\n" +
+            "You can execute commands from main menu on the left or by typing a command:\n\n" +
+            "Type /start to see welcome message\n\n" +
+            "Type /my_data to see data stored about yourself\n\n" +
+            "Type /delete_data to delete stored user data\n\n" +
+            "Type /help to see message again\n\n" +
+            "Type /settings to users settings";
+
+    public TelegramBot(BotConfig config) {
+        this.config = config;
+        List<BotCommand> listOfCommands = new ArrayList<>();
+        listOfCommands.add(new BotCommand("/start", "get a welcome message"));
+        listOfCommands.add(new BotCommand("/my_data", "get your data stored"));
+        listOfCommands.add(new BotCommand("/delete_data", "delete my data"));
+        listOfCommands.add(new BotCommand("/help", "info how to use this bot"));
+        listOfCommands.add(new BotCommand("/settings", "set you preferences"));
+        try {
+            this.execute(new SetMyCommands(listOfCommands, new BotCommandScopeDefault(), null));
```

Comment:
Usually doing something more complex than variables initialization in constructor is bad practice. Consider creating a dedicated method which is called after the class is initialized by a caller.